### PR TITLE
MPP-3688, MPP-3689 and MPP-3690 strings (Add-on phone masking)

### PR DESCRIPTION
**Pull request by rafeerahman**
Strings for https://github.com/mozilla/fx-private-relay-add-on/pull/553

### CODE REVIEW

File: en/messages.json [hunk 1]
```diff
@@ -594,5 +594,79 @@
     "popupHolidayPromo2023CTA": {
         "message": "Get 1 year of Relay Premium",
         "description": "DO NOT TRANSLATE \"Relay Premium\"."
+    },
+    "popupPhoneMasksMetaForwardingDescription": {
+        "message": "Relay is currently forwarding all phone calls and text messages to your true phone number.",
+        "description": "DO NOT TRANSLATE \"Relay\"."
+    },
+    "popupPhoneMasksMetaBlockingDescription": {
+        "message": "Relay is blocking all phone calls and text messages—you will not receive anything from your phone number mask.",
```
Comment (flodolo) [inline en/messages.json:603]:
```suggestion
        "message": "Relay is blocking all phone calls and text messages — you will not receive anything from your phone number mask.",
```

https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o#section-hyphens-and-dashes-c9

File: en/messages.json [hunk 2]
```diff
@@ -594,5 +594,79 @@
...
+        "description": "DO NOT TRANSLATE \"Relay\"."
+    },
+    "popupPhoneMasksMetaForwardTo": {
+        "message": "Forward to:"
+    },
+    "popupPhoneMasksMetaRegistered": {
+        "message": "Registered:"
```
Comment (flodolo) [inline en/messages.json:610]: What does "Registered" means in this context? Is there a screenshot? Once clarified, this string needs a comment (description) to clarify it.

**Review by flodolo — CHANGES_REQUESTED** [review]

File: en/messages.json (hunk 2 above)
Comment (rafeerahman) [reply en/messages.json]: Added descriptions. Here is some context, see "registered" near the bottom.
302500456-363be5ca-eebd-442a-9b01-74d2d67dab69

Comment (flodolo) [reply en/messages.json]: Thanks, this is helpful. The image say Emails/Phones, I assume that changed to Email/Phone now, given the strings?

I have a feeling you're going to have issues with a 2-column layout, some languages will have longer translations, some potentially even single words that are longer than the available space. Was the layout tested with a test much longer, e.g. twice the length of English??

Comment (rafeerahman) [reply en/messages.json]: Correct, that was an older screenshot, it is "Email" and "Phone" now.

Great catch, did some testing now and it did break the layout. Fixed it [here](https://github.com/mozilla/fx-private-relay-add-on/pull/553/commits/cc8836ab914ef25bd336e475394d709c0f672156), it now wraps the text incase translated strings are long.

**Review by flodolo — APPROVED** [review]

**Pull request merged**